src/views/LayoutEntry.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'react-flexbox-grid';
 import HeaderInitial from '../components/HeaderInitial';
 import Footer from '../components/Footer';
 
-const LayoutEntry = ({ children }) => (<div>
-  <HeaderInitial />
-  <div>
-    {children}
-  </div>
-  <Footer />
-</div>);
+const LayoutEntry = ({ children }) =>
```

Comment:
This component has two functions:

* It wraps the children in a header and footer.
* It (tries to) align the children in three columns.

I think these are two separate functions that should not be mixed in one component. This component should focus purely on the header/footer (like the `Layout` component it is named after).

The layout should go directly into the `LoginView`. We don't need a special component for a three column layout, react-flexbox-grid makes it easy enough.

(Now, if it turns out that we are repeating the same layout in many views, we can consider factoring it out. But layouts are so tightly integrated with their views that this will be hard, and should only be done if there is a significant benefit, like a big reduction in repeated code. The important thing is, we don't do this now.)